components/PlantInfo.vue

```diff
@@ -23,15 +23,16 @@
     </v-btn>
     <v-card flat>
       <v-carousel
+        v-if="plantInfo.images.length > 0"
```

Comment:
use v-show instead of v-if. v-if should only be used if there's multiple conditions. Pls v-show is faster